mlflow/fastai.py

```diff
@@ -43,6 +43,18 @@
 FLAVOR_NAME = "fastai"
 
 
+def _get_default_pip_requirements(include_cloudpickle=False):
```

Comment:
```suggestion
def _get_default_pip_requirements(include_cloudpickle=False):
     """
     Returns a default set of pip requirements of this flavor.
     """
```

Reply:
docstring may clarify what this function returns.

Reply:
How about 


```
:return: A list of default pip requirements for MLflow Models produced by this flavor.
             Calls to :func:`save_model()` and :func:`log_model()` produce a pip environment
             that, at minimum, contains these requirements.
```

Reply:
When we implement pip dependency inference, can we also update `get_default_conda_env()` to indicate that the default conda env is a subset of the full set of dependencies generated by `save_model()` and `log_model()`? - we can do this as part of a follow-up PR.

Reply:
> How about ...

Sounds good!

Reply:
got it!